themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -107,6 +107,12 @@
     ?>
     <?=$this->cookieConsent()->render()?>
     <?=$this->headScript() ?>
+    <?php if (isset($lightboxParent)): ?>
```

Comment:
I'd use `!empty()` instead of `isset()` so that e.g. an empty string is ignored.